package.json

```diff
@@ -5,9 +5,11 @@
   "main": "index.js",
   "scripts": {
     "build": "razzle build",
+    "build-storybook": "build-storybook -s public -o .out",
     "cypress": "cypress run",
     "cypress:interactive": "cypress open",
     "cypress:storybook": "cypress run --project ./.storybook/.",
+    "deploy-storybook": "storybook-to-ghpages",
```

Comment:
When I run this script locally, it commits to the gh-pages branch with these details
```
   executing: git init
   executing: git config user.name "GH Pages Bot"
   executing: git config user.email "hello@ghbot.com"
   executing: git config commit.gpgsign false
```

I am not sure that we want anyone on any branch to update Storybook with what they have available locally.  This could cause confusion if the Storybook is 'broken' with work on someone's branch.